nextcord/client.py

```diff
@@ -24,51 +24,55 @@
 
 from __future__ import annotations
 
+import aiohttp
```

Comment:
mmmh pep8 imports remember

Reply:
Not sure what you mean? `aiohttp` comes before `asyncio` alphabetically.

Reply:
aiohttp is not stdlib